README.md

```diff
@@ -1 +1,105 @@
-# TDD_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+# TDD project
+
+## Description 🏗️
+Practice Test Driven Development by writing tests before code that makes it work
+
+## Built With
+
+- Ruby
+
+## Getting Started 🏁
+
+### Prerequisites 📜
+
+You will be needing:
+
+- A terminal terminal
+- A code editor
+- Ruby (follow the instruction based on your OS)
+```bash
+https://www.ruby-lang.org/en/documentation/installation/
+```
+
+### Setup
+
+### Clone this repository
+
+```bash
+git clone https://github.com/ogagaoghene/TDD_Project.git
+```
+### Move into the directory with
+
+```bash
+cd oop_school_library_ruby

```

Comment:
- [ ] [OPTIONAL] Here kindly check the readme file & correct the file name.

Reply:
I will do as requested. Thanks alot